### DISCUSSION TITLE
Can an extension provide a localized version of README.md?

### DISCUSSION THREAD

**Discussion by jonathanyeung** [microsoft/vscode-discussions#1359] [Extension Development QnA] [answered]
Hi,

Is it possible for an extension to provide a localized version of the README.md file that appears when viewing an extension in the 'Extension Tab'?  On top of that, are there international versions of the extensions marketplace website https://marketplace.visualstudio.com/vscode ?

Thanks!

**Comment by isidorn** [comment, marked as answer]
This is currently not possible. And is a fair request against the marketplace.

fyi @seaniyer as the Marketplace PM
